routes/users.js

```diff
@@ -0,0 +1,31 @@
+
+let users = require ('../database/users.js');
+const express = require('express');
+const router = express.Router();
+
+
+
+  
+  router.post('/', (req, res) => {
+    users.push(req.body);
+    console.log(req.body);  
+    res.send('account created');
+  });
+
+  router.put('/:id', (req, res) => {
+      users[req.params.id] = req.body;
+    res.send('account edited');
+  });
+
+  router.get('/:id', (req, res) => {
+    let account = users[req.params.id];      
+    res.json({account});  
+    
+  });
+  
+  router.delete('/:id', (req, res) => {
+    users.splice(req.params.id,1);  
```

Comment:
minor issue: spaces around comma ','
you should use eslint and right configuration: for examle: eslint airbnb-base